core/shared/src/main/scala/zio/Cause.scala

```diff
@@ -264,7 +294,7 @@ sealed trait Cause[+E] extends Product with Serializable { self =>
 
     def causeToSequential(cause: Cause[Any], maybeData: Option[Data]): Sequential =
       cause match {
-        case Empty => Sequential(Nil)
+        case Empty | Meta(_, Data(_, true)) => Sequential(Nil)
```

Comment:
I think we want to keep the logic for pretty printing a `Cause` separate from whether the `DefaultRuntime` should report a failure. So I would just ignore `silent` for purpose of pretty printing.